src/ping.js

```diff
@@ -38,6 +38,7 @@ function ping (options) {
       reject(err)
     })
     client.once('server_info', function (packet) {
+      packet.response = packet.response.replace(/\n/g, '\\n')
```

Comment:
Can you put it into a function like parseLenientJson ?

Apparently this is not json